src/index.js

```diff
@@ -0,0 +1,131 @@
+// ------------- Wave 2 ----------------------
+const state = {
+  temp: 60,
+  city: 'Seattle',
+  sky: '☁️',
+};
+// 1. increase temp
+const addDegree = (event) => {
+  state.temp += 1;
+  console.log('state temp', state.temp);
+  const tempContainer = document.querySelector('#degrees');
```

Comment:
Use `getElementById` instead if you're simply searching for an element using its id.